lib/ansible/modules/yum_repository.py

```diff
@@ -109,6 +109,13 @@
     description:
       - A URL pointing to the ASCII-armored GPG key file for the repository.
       - It can also be a list of multiple URLs.
+  module_hotfixes:
+    description:
+      - Disable module RPM filtering and make all RPMs from the repository
+        available. The default is C(False).
+    version_added: '2.10'
```

Comment:
This will need to be changed to 2.11 once we branch for the 2.10 release.
```suggestion
    version_added: '2.11'
```

This can be changed now, but it will fail sanity tests until we update them.

Reply:
That is so unfortunate, as this feature is really needed for centos 8, which is here for a while...